ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/EngineGetPayloadResultWithdraws.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.results;
+
+import org.hyperledger.besu.ethereum.core.BlockHeader;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.apache.tuweni.bytes.Bytes32;
+
+@JsonPropertyOrder({
+  "executionPayload",
+  "blockValue",
+})
+public class EngineGetPayloadResultWithdraws extends AbstractEngineGetPayloadResult {
```

Comment:
This probably contradicts what I said yesterday, but I'm thinking V2 in the name makes more sense than Withdrawals, although I still don't think V2 should leak further than the API level.

Although this object will end up have Withdrawals in it, including withdrawals could and maybe will be data driven (due to presence of Optional withdrawals). 
In this PR, the trigger for this different class structure to be built is really blockValue, but I don't think EngineGetPayloadResultBlockValue makes sense either. 

Probably better we keep the code close to the spec in this regard, at least for now until it's clear how the API will evolve with V3